specification/communication/data-plane/JobRouter/preview/2022-07-18-preview/communicationservicejobrouter.json

```diff
@@ -980,11 +988,25 @@
           },
           {
             "in": "query",
-            "name": "maxpagesize",
+            "name": "scheduledBefore",
+            "description": "(Optional) If specified, filter on jobs that was scheduled before or at given timestamp. Range: (-Inf, scheduledBefore]",
```

Comment:
Don't add "(Optional)" in the description as it is redundant with the `required` attribute of the parameter.

Reply:
This has been removed